hbase-http/src/main/java/org/apache/hadoop/hbase/http/HttpServer.java

```diff
@@ -152,6 +152,9 @@ public class HttpServer implements FilterContainer {
     "hbase.security.authentication.ui.config.protected";
   public static final String HTTP_UI_NO_CACHE_ENABLE_KEY = "hbase.http.filter.no-store.enable";
   public static final boolean HTTP_PRIVILEGED_CONF_DEFAULT = false;
+  public static final String HTTP_PRIVILEGED_METRICS_KEY =
```

Comment:
Can we have a test case similar to the hadoop PR? Else it's difficult to ensure this doesn't break in future.

Reply:
I got it, thanks. I will prepare with that.

Reply:
I'm trying to add a test case but I encountered a problem. In order to use "hbase.security.authentication.spnego.kerberos.endpoint.whitelist", HBase should use the hadoop which has HADOOP-18666.
Officially, HADOOP-18666 is committed hadoop >= 3.4.0.

Reply:
Oh oh, in that case this feature won't even work until we have hadoop-3.4 right/ Maybe we park this PR for later then?

Reply:
I think so.
Let's postpone this PR until hbase uses hadoop with HADOOP-18666.